taxi/tests.py

```diff
@@ -1,3 +1,27 @@
 from django.test import TestCase
+from django.contrib.auth import get_user_model
+from taxi.models import Manufacturer, Car
 
-# Create your tests here.
+
+class ModelsTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="BMW", country="Germany")
+        self.assertEqual(str(manufacturer), f"{manufacturer.name} {manufacturer.country}")
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="test",
+            password="test123",
+            first_name="tester",
+            last_name="test",
+            license_number="AMD12345"
+        )
+        self.assertEqual(str(driver), f"{driver.username} ({driver.first_name} {driver.last_name})")
+
+    def test_car_str(self):
+        manufacturer = Manufacturer.objects.create(name="BMW", country="Germany")
+        car = Car.objects.create(
+            model="r6",
+            manufacturer=manufacturer,
+        )
+        self.assertEqual(str(car), car.model)
```

Comment:
The test for the `Car` model's string representation only checks the model name. Ensure this matches the intended behavior of the `__str__` method in the `Car` model. If the `__str__` method is supposed to include more details, adjust the test accordingly.